### PR TITLE
Create childOrgEnrollment

### LINKED ISSUE #39: childOrgEnrollment
Standard: 3
Sequence: 3
Contract Name: childOrgEnrollment
Problem Statement: https://github.com/Aniket-Engg/solidity-school/tree/master/std-3/3_childOrgEnrollment

Solve the problem statement.

Contribution guidelines: https://github.com/Aniket-Engg/solidity-school/wiki/Contribution-Guidelines

### PR DESCRIPTION
**Pull request by amar-exe**
fixes #39 

Please check if everything is as intended in the task, if it is not LMK and I will fix it,

best regards

### CODE REVIEW

**Review by saba8814 — APPROVED** [review]
LGTM!

File: std-3/childOrgEnrollment [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  function enroll(address _employeeAddress, string memory _name, uint _age, string memory _department, string memory _childOrgName) public virtual {
+    employees[_employeeAddress] = EmployeeDetails(_name, _age, _department, _childOrgName);
+  }
+}
+
+contract NewCompany is Enrollment {
+
+  // child contract that overrides the enroll method to include child organization specific information
+  function enroll(string memory _name, uint _age, string memory _department, string memory _childOrgName) public {
```
Comment (Aniket-Engg) [inline std-3/childOrgEnrollment:25]: This is not actually overriding the base `enroll` method. Also, the overriding function must use the `override` keyword in the function header. Check the hint given in problem statement here: https://github.com/Aniket-Engg/solidity-school/tree/master/std-3/3_childOrgEnrollment

**Review by Aniket-Engg — CHANGES_REQUESTED** [review]

**Comment by Aniket-Engg** [comment]
@amar-exe can you consider the requested changes?

File: std-3/childOrgEnrollment [hunk 2]
```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.17;
+
+contract Enrollment {
```
[the remark this answers is not in the document]
Comment (amar-exe) [reply std-3/childOrgEnrollment:5]: Changed name

File: std-3/childOrgEnrollment [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  }
+
+  mapping (address => EmployeeDetails) employees;
+
+  function enroll(address _employeeAddress, string memory _name, uint _age, string memory _department, string memory _childOrgName) public virtual {
+    employees[_employeeAddress] = EmployeeDetails(_name, _age, _department, _childOrgName);
+  }
+}
+
+contract NewCompany is Enrollment {
```
[the remark this answers is not in the document]
Comment (amar-exe) [reply std-3/childOrgEnrollment:22]: Updated

**Comment by amar-exe** [comment]
@Aniket-Engg I implemented the changes, please check if everything is correct

**Review by Aniket-Engg — APPROVED** [review]

**Comment by amar-exe** [comment]
@Aniket-Engg do you have any extra assignments or things that need to be done?

**Comment by saba8814** [comment]
@Aniket-Engg I'd be happy to help too, You can assign me some tasks too.

**Pull request merged**